website/docs/r/cdn_frontdoor_secret.html.markdown

```diff
@@ -12,7 +12,11 @@ Manages a Front Door (standard/premium) Secret.
 
 ## Required Key Vault Permissions
 
-!>**IMPORTANT:** You must add an `Access Policy` to your `azurerm_key_vault` for the `Microsoft.Azure.Cdn` Enterprise Application Object ID.
+!>**IMPORTANT:** You must add an `Access Policy` to your `azurerm_key_vault` for the `Microsoft.AzurefrontDoor-Cdn` Enterprise Application Object ID.
+
+This can be created by running Az Powershell command like this:
+
+```New-AzADServicePrincipal -ApplicationId "205478c0-bd83-4e1b-a9d6-db63a3e1e1c8"```
```

Comment:
Could we replace the appID with a placeholder e.g. 
```suggestion
```New-AzADServicePrincipal -ApplicationId "00000000-0000-0000-0000-000000000000"```
```